keys/keys.go

```diff
@@ -162,6 +170,28 @@ func (m *keyManager) recoveryFromPrivateKey(privateKey string) error {
 	return nil
 }
 
+func (m *keyManager) recoveryFromLedgerKey(path ledger.DerivationPath) error {
+	if ledger.DiscoverLedger == nil {
+		return fmt.Errorf("no Ledger discovery function defined")
```

Comment:
can this error be more user friendly? we should list what is really missed or what user should do.